metricbeat/module/zookeeper/connection/connection.go

```diff
@@ -61,17 +64,17 @@ func New(base mb.BaseMetricSet) (mb.MetricSet, error) {
 func (m *MetricSet) Fetch(reporter mb.ReporterV2) error {
 	outputReader, err := zookeeper.RunCommand("cons", m.Host(), m.Module().Config().Timeout)
 	if err != nil {
-		return errors.Wrap(err, "'cons' command failed")
+		return errors.Unwrap(fmt.Errorf(err.Error(), "'cons' command failed: %v"))
```

Comment:
Umm, why doing these unwraps? I think this is line is equivalent to `return err.Error()`, losing the context information.

If you want to migrate to `fmt.Errorf`, I think you should do something like this:
```suggestion
		return fmt.Errorf("'cons' command failed: %w", err))
```

Reply:
Removed it.!!!